designsafe/apps/accounts/views.py

```diff
@@ -35,6 +35,11 @@ def manage_profile(request):
     """
     django_user = request.user
     user_profile = TASUser(username=request.user.username)
+    try:
+        ds_profile = DesignSafeProfile.objects.get(user__id=django_user.id)
+    except DesignSafeProfile.DoesNotExist:
+        logout(request)
```

Comment:
I'm not understanding why we need to logout a user here. Can you explain?
Thanks.

Reply:
This comes directly from the old `manage_pro_profile` profile view that is now combined with `manage_profile`. I'm not sure what the rationale is but I didn't want to change the logic.